linera-views/src/views/unit_tests/views.rs

```diff
@@ -437,7 +437,7 @@ async fn test_reentrant_collection_view_has_pending_changes_after_try_load_entry
     Ok(())
 }
 
-/// Check if a acquiring multiple write-locks to sub-views causes the collection to have pending
+/// Check if an acquiring multiple write-locks to sub-views causes the collection to have pending
```

Comment:
I think it's actually supposed to be this:
```suggestion
/// Check if acquiring multiple write-locks to sub-views causes the collection to have pending
```